newrelic/structures_newrelic_one_dashboard.go

```diff
@@ -287,6 +287,7 @@ func expandDashboardPageInput(d *schema.ResourceData, pages []interface{}, meta
 			}
 		}
 		if widgets, ok := p["widget_line"]; ok {
+			widgets = widgets.(*schema.Set).List()
```

Comment:
(a)
This change, paired with `schema.TypeSet` in `widget_line` fix the issue seen with the drift. If the configuration and the state differ in nothing apart from the order of line widgets, no drift is shown.

But now, I run into a new problem where a drift is shown across all line widgets, even if only one attribute's value in one line widget is changes.